### PR TITLE
TIP-307: fix download pdf issue when using PHP7

### PR DESCRIPTION
**Pull request by nidup**
Following issue happens when using PHP7 (still experimental support for now).

php7-issue

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | N
| Added Behats                      | N
| Changelog updated                 | N
| Review and 2 GTM                  | ?
| Micro Demo to the PO (Story only) | N
| Migration script                  | N
| Tech Doc                          | N

### CODE REVIEW

**Comment by loicguillois** [comment]
GTM

**Comment by willy-ahva** [comment]
GTM

**Comment by phaseinducer** [comment]
GTM

File: src/Pim/Bundle/PdfGeneratorBundle/Resources/views/Product/renderPdf.html.twig [hunk 1]
```diff
@@ -107,7 +107,11 @@
                                     {% set manualHeight = false %}
                                 {% else %}
                                     {% set content = product.getValue(attribute.code, locale, scope) %}
-                                    {% set manualHeight = attribute.label|length > (content.__toString()|length / 3) %}
+                                    {% if content != null %}
+                                        {% set manualHeight = attribute.label|length >(content.__toString()|length / 3) %}
+                                    {% else %}
+                                        {% set manualHeight = true %}
```
Comment (SamirBoulil) [inline src/Pim/Bundle/PdfGeneratorBundle/Resources/views/Product/renderPdf.html.twig]: is it normal that we set a boolean when the variable type should be float ?

Comment (nidup) [inline src/Pim/Bundle/PdfGeneratorBundle/Resources/views/Product/renderPdf.html.twig]: `attribute.label|length > (content.__toString()|length / 3)` returns a boolean

Comment (SamirBoulil) [inline src/Pim/Bundle/PdfGeneratorBundle/Resources/views/Product/renderPdf.html.twig]: :) sorry

Comment (nidup) [inline src/Pim/Bundle/PdfGeneratorBundle/Resources/views/Product/renderPdf.html.twig]: no pb :eyes: of the :tiger:

**Pull request merged**